packages/thirdweb/src/wallets/connection/autoConnectCore.ts

```diff
@@ -183,13 +183,18 @@ const _autoConnectCore = async ({
         wallet,
         client: props.client,
         lastConnectedChain,
-        authResult: result?.authResult,
+        authResult: urlToken?.authResult,
       });
       manager.addConnectedWallet(wallet);
     } catch {
       // no-op
     }
   }
+
+  // Auto-login with SIWE
+  if (urlToken && activeWallet && props.siweAuth?.requiresAuth) {
```

Comment:
You don't need to invoke this? I thought it was a function

Reply:
We should do the same logic that would trigger the sign in button

Reply:
Not a function, it just checks if they're using SIWE